app/src/main/java/com/infomaniak/mail/data/cache/RealmMigrations.kt

```diff
@@ -73,3 +75,16 @@ private fun MigrationContext.keepDefaultValuesAfterSixthMigration() {
         }
     }
 }
+
+// Migrate from version #19
+private fun MigrationContext.keepDefaultValuesAfterNineteenthMigration() {
+    if (oldRealm.schemaVersion() <= 19L) {
+        enumerate(className = "Folder") { oldObject: DynamicRealmObject, newObject: DynamicMutableRealmObject? ->
+            newObject?.apply {
+                // Add property with default value
+                set(propertyName = "isDisplayed", value = true)
+            }
+        }
+    }
+}
```

Comment:
The new spec says to always show the folder to simplify things out. No need to hide it anymore

Reply:
Do we really want to keep all the time a Folder tied to a feature that probably not that much of users will ever use ? :/